tools/devops/build-samples.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash -eux
 
+cd "$(dirname "${BASH_SOURCE[0]}")/../.."
+
 # we want verbose output from mtouch and mlaunch
 echo 123456789 > ~/.mtouch-verbosity
 echo 123456789 > ~/.mlaunch-verbosity
```

Comment:
* is `mlaunch` used to build samples ?
* maybe use the `[MTOUCH|MMP]_ENV_OPTIONS` variable (to centralize where to make changes) ?
* if not then maybe (not sure if supported) the same for `mmp` ?

Reply:
No, mlaunch is not used. I removed these two lines.

I added `-vvvv` to `[MTOUCH|MMP]_ENV_OPTIONS]` instead.